### DISCUSSION TITLE
Path autocompletion has unneeded forward slash

### DISCUSSION THREAD

**Discussion by vdualb** [helix-editor/helix#9576] [General]
### Summary

I have this folder structure:
```
(.venv) ⬢ hxtest tree                
.
├── some_dir
│   └── 1file
└── test.py

2 directories, 2 files
```
content of `test.py`:
```
(.venv) ⬢ hxtest cat test.py 
a = open('so')
```
After navigating to second single quote, typing `me` and pressing down button, path is autocompleted to this `a = open('some_dir\/')` instead of `a = open('some_dir/')`

### Reproduction Steps

 

I tried this:

1. Recreate folder structure mentioned above
2. `helix test.py`
3. Type `me` after `so`
4. Press down button to autocomplete path

I expected this to happen:
 `a = open('some_dir\/')`

Instead, this happened:
`a = open('some_dir/')`

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux

### Terminal Emulator

Prompt a79a6c

### Installation Method

arch package

### Helix Version

23.10

**Comment by pascalkuthe** [comment]
Helix doesn't provide path completions. I guess they may be provided by your python language server in that case you should report this issue to them.

**Reply by vdualb** [reply]
Looks like this issue in pylsp: https://github.com/python-lsp/python-lsp-server/issues/373 , but it was fixed and I should be running fixed version.
